resources/events.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Events Calendar
+order: 1
+---
+
+# Events Calendar
+<iframe id="open-web-calendar" 
+    style="background:url('https://raw.githubusercontent.com/niccokunzmann/open-web-calendar/master/static/img/loaders/circular-loader.gif') center center no-repeat;"
+    src="https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fics.teamup.com%2Ffeed%2Fks3qwzicncjfkkeyq6%2F8703308.ics"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation"
```

Comment:
A lot of permissions. Can we at least remove `allow-same-origin`?

Reply:
```Access to font at 'https://open-web-calendar.herokuapp.com/css/dhtmlx/dhtmlxscheduler_material/fonts.googleapis.com/KFOlCnqEu92Fr1MmEU9fBBc4.woff2' from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.```
Got this error.